aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -166,10 +172,10 @@ class QLinearInt8 final : public torch::OperatorKernel {
 
 static auto registry =
     torch::RegisterOperators()
-        .op("quantized::linear(Tensor X, Tensor W_prepack, Tensor? b, float Y_scale_i, int Y_zero_point_i) -> Tensor Y",
+        .op("quantized::linear(Tensor X, Tensor W_prepack, float Y_scale_i, int Y_zero_point_i) -> Tensor Y",
```

Comment:
@jamesr66a - just to confirm, we don't ship this anywhere so it's fine to change the signature, right?

Reply:
Yeah I *believe* it's okay but if we can pull operator stats from somewhere to confirm that would be safest